src/index.html

```diff
@@ -3,12 +3,93 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no,
+          initial-scale=1.0, maximum-scale=1.0,
+          minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <img class="headerImg"
+          src ="images/logo.png"
+          alt="logo"
+          href="#home"
+        >
+        <nav class="nav">
+          <ul class="list"></ul>
+          <li class="liItem">
+            <a
+              class="nav_link"
+              href="#"
+            >
```

Comment:
2 attributes go in 1 line
```suggestion
            <a class="nav_link" href="#">
```